### PR TITLE
Release portworx-data-science-engine 1.3-1.0.0 (automated commit)



### PR DESCRIPTION
**Pull request by piyush-nimbalkar**
Release portworx-data-science-engine 1.3-1.0.0 (automated commit)

Description:
Source URL: https://px-dcos-dev.s3.amazonaws.com/autodelete7d/portworx-data-science-engine/20191127-183952-EqW83qvlTvJoC5aX/stub-universe-portworx-data-science-engine.json

Changes between revisions -1 => 0:
4 files added: [package.json, resource.json, marathon.json.mustache, config.json]
0 files removed: []
0 files changed:

```
```

### CODE REVIEW

File: repo/packages/P/portworx-data-science-engine/0/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.12",
+  "name": "portworx-data-science-engine",
+  "version": "1.3-1.0.0",
+  "scm": "https://github.com/mesosphere/jupyter-service",
+  "maintainer": "support@portworx.com",
+  "website": "https://docs.mesosphere.com/services/data-science-engine",
+  "description": "Portworx Data Science Engine provisions Jupyter Notebooks, an open-source web application that allows you to create and share documents that contain live code, equations, visualizations and narrative text.\nUses include: data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning, and much more.",
```
Comment (disrani-px) [inline repo/packages/P/portworx-data-science-engine/0/package.json:9]: The description needs to have some of the same language as the base package: https://github.com/portworx/jupyter-service/blob/master/universe/package.json#L9

File: repo/packages/P/portworx-data-science-engine/0/package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.12",
+  "name": "portworx-data-science-engine",
+  "version": "1.3-1.0.0",
```
Comment (disrani-px) [inline repo/packages/P/portworx-data-science-engine/0/package.json:5]: Don't need the 1.3 prefix since this isn't based on the sdk

**Review by disrani-px — CHANGES_REQUESTED** [review]

File: repo/packages/P/portworx-data-science-engine/0/package.json (hunk 1 above)
Comment (piyush-nimbalkar) [reply repo/packages/P/portworx-data-science-engine/0/package.json:9]: That's from master. The released tag had the above description. 
But looks like the actual package has description from master. I will update it.

**Comment by piyush-nimbalkar** [comment]
Created new PR - https://github.com/portworx/universe/pull/159

**Pull request closed without merging**